.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,13 +2,13 @@ Thanks for submitting a PR, your contribution is really appreciated!
 
 Here's a quick checklist that should be present in PRs:
 
-- [ ] Target: for bug or doc fixes, target `master`; for new features, target `features`;
+- [ ] Add a new news fragment into the changelog folder
+  * name it `$issue_id.$type` for example (588.bug)
+  * if you dont have an issue_id change it to the pr id after creating the pr
+  * type is one of `removal`, `feature`, `bugfix`, `vendor`, `doc`, `trivial`
```

Comment:
Final nitpick before merging, I think we should add:

```
* Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."
```

To ensure a consistent-looking changelog.

Also the bullet points in here themselves could use the same treatment. 😉